genesis.json

```
@@ -0,0 +1,560 @@
+{
+  "genesis_time": "2021-01-19T18:53:31.685Z",
+  "chain_id": "sommelier",
```

this needs to have the epoch number for IBC compat
```suggestion
  "chain_id": "sommelier-1",
```